indico/modules/events/editing/controllers/backend/management.py

```diff
@@ -108,37 +107,36 @@ def _process_args(self):
         self.editable_type = EditableType[request.view_args['type']]
 
     def _process_GET(self):
-        return jsonify(editing_settings.get(self.event, self.editable_type.name + '_review_conditions').items())
+        query = (
+            EditingReviewCondition.query.with_parent(self.event)
+            .filter_by(type=self.editable_type)
+            .order_by(EditingReviewCondition.id)
+            .all()
+        )
+        return jsonify([[cond.id, sorted([ft.id for ft in cond.file_types])] for cond in query])
 
     @use_rh_kwargs(EditingReviewConditionArgs)
     def _process_POST(self, file_types):
-        new_conditions = editing_settings.get(self.event, self.editable_type.name + '_review_conditions')
-        new_conditions[unicode(uuid4())] = file_types
-        editing_settings.set(self.event, self.editable_type.name + '_review_conditions', new_conditions)
+        full_file_types = set(EditingFileType.query.with_parent(self.event).filter(EditingFileType.id.in_(file_types)))
+        create_new_review_condition(self.event, self.editable_type, full_file_types)
         return '', 204
 
 
 class RHEditingEditReviewCondition(RHEditingManagementBase):
     def _process_args(self):
         RHEditingManagementBase._process_args(self)
         self.editable_type = EditableType[request.view_args['type']]
-        uuid = request.view_args['uuid']
-        conditions = editing_settings.get(self.event, self.editable_type.name + '_review_conditions')
-        self.uuid = unicode(uuid)
-        if self.uuid not in conditions:
-            raise NotFound
+        id_ = request.view_args['condition_id']
+        self.condition = (EditingReviewCondition.query.with_parent(self.event).filter_by(id=id_).first_or_404())
```

Comment:
no need for the `()` around the whole thing since it's all on one line.

alternatively, go for this instead:

```python
        self.condition = (EditingReviewCondition.query
                          .with_parent(self.event)
                          .filter_by(id=request.view_args['condition_id'])
                          .first_or_404())
```